torch/csrc/jit/passes/quantization.cpp

```diff
@@ -39,40 +39,42 @@ int groups, bool benchmark, bool deterministic, bool cudnn_enabled) -> Tensor"
   return quantnodeLookup.find(n);
 }
 
-void insertQuantNodeParams(Node* quant, std::tuple<float, int> qparam) {
+void insertQuantNodeParams(Node* quant, float scale, int zero_point) {
```

Comment:
This seems like a refactoring that doesn't change what the function is doing - please split such things into separate PRs. It makes reviewing and potential future debugging easier.